addon/services/file-manager.js

```diff
@@ -14,6 +14,27 @@ export default Ember.Service.extend({
     store: Ember.inject.service(),
 
     /**
+     * Get a URL to download the given file.
+     *
+     * @method getDownloadUrl
+     * @param {file} file A `file` model
+     * @param {Object} [options] Options hash
+     * @param {Object} [options.query] Key-value hash of query parameters to
+     * add to the URL.
+     * @param {Object} [options.query.version] `file-version` ID
+     * @return {String} Download URL
+     */
+    getDownloadUrl(file, options = {}) {
+        let url = file.get('links.download');
+
+        if (file.isFolder) {
+            options.query.zip = '';
```

Comment:
Clicking on the download link in nodes/detail/files/provider/file_id fails here.  The versionID is undefined instead of being the latest version.  Downloading in the revisions view works great, however.

![screen shot 2016-06-17 at 2 27 32 pm](https://cloud.githubusercontent.com/assets/9755598/16160736/b7577608-3497-11e6-922f-30e91d54fd3c.png)